policy/modules/kernel/filesystem.fc

```diff
@@ -20,6 +20,8 @@
 /sys/fs/pstore	-d	gen_context(system_u:object_r:pstore_t,s0)
 /sys/fs/pstore/.*	<<none>>
 
+/sys/kernel/tracing(/.*)?	<<none>>
```

Comment:
Maybe also add the entry `/sys/kernel/tracing                -d      gen_context(system_u:object_r:tracefs_t,s0)` ?